src/app.js

```diff
@@ -1,21 +1,32 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const fs = require('fs');
+const path = require('path');
+
+const file = process.argv[2];
+const newFile = process.argv[3];
+
+let newFilePath = newFile;
+
+if (newFile[newFile.length - 1] === '/') {
```

Comment:
```suggestion
if (newFile.at(-1) === '/') {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at